### ISSUE TITLE
How to get everypixel's logits info at Paddle Inference C++

### ISSUE THREAD

**Issue by zrnwin** [PaddlePaddle/PaddleSeg#1977]
Hi:
I'm using PaddleSeg recently, in Paddle Inference C++ file:PaddleSeg/blob/release/2.4/deploy/cpp/src/test_seg.cc, I found that I can only get pred after argmax, but I need every pixel's logits info.Can you provide the corresponding method. Thank you very much for your continued efforts.
My PaddleSeg version: release 2.4
Here is the get out part in :test_seg.cc

 // Get output
  auto output_names = predictor->GetOutputNames();
  auto output_t = predictor->GetOutputHandle(output_names[0]);
  std::vector<int> output_shape = output_t->shape();
  int out_num = std::accumulate(output_shape.begin(), output_shape.end(), 1,
                                std::multiplies<int>());
  std::vector<int64_t> out_data(out_num);
  output_t->CopyToCpu(out_data.data());

  // Get pseudo image
  std::vector<uint8_t> out_data_u8(out_num);
  for (int i = 0; i < out_num; i++) {
    out_data_u8[i] = static_cast<uint8_t>(out_data[i]);
  }
  cv::Mat out_gray_img(output_shape[1], output_shape[2], CV_8UC1, out_data_u8.data());
  cv::Mat out_eq_img;
  cv::equalizeHist(out_gray_img, out_eq_img);
  cv::imwrite("out_img.jpg", out_eq_img);

**Comment by juncaipeng** [comment]
In exporting inference model ([doc](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.4/docs/model_export.md)), you should add '--without_argmax', so the output of the inference model is the logits of each class (dims is n*c*h*w).

**Comment by zrnwin** [comment]
> In exporting inference model ([doc](https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.4/docs/model_export.md)), you should add '--without_argmax', so the output of the inference model is the logits of each class (dims is n_c_h*w).

Thank you for your reply.
